packages/p2p/src/stores/general-store.js

```diff
@@ -530,6 +530,7 @@ export default class GeneralStore extends BaseStore {
         this.setActiveIndex(0);
         this.external_stores?.notifications.refreshNotifications();
         this.external_stores?.notifications.filterNotificationMessages();
+        this.setIsLoading(true);
```

Comment:
do we need this here if unmount p2p page?

Reply:
@farrah-deriv 
yes because when we switched back let say to Transfer tab in cashier our `is_loading` state in store equals `false`, so when we again try to open p2p tab since it false we will see content blinking. in order to prevent this we are setting `is_loading` back to `true` to avoid blinking when we coming back to p2p again.